WordPress/src/main/java/org/wordpress/android/models/Blog.java

```diff
@@ -497,7 +499,30 @@ public void setPlanID(long planID) {
     public String getPlanShortName() {
         return StringUtils.notNullStr(planShortName);
     }
+
     public void setPlanShortName(String name) {
         this.planShortName = StringUtils.notNullStr(name);
     }
+
+    public String getCapabilities() {
+        return capabilities;
+    }
+
+    public void setCapabilities(String capabilities) {
+        this.capabilities = capabilities;
+    }
+
+    public boolean checkCapability(Capability capability) {
+        // If a capability is missing it means the user don't have it.
+        if (capabilities == null || capabilities.isEmpty() || capability == null) {
+            return false;
+        }
+        try {
+            JSONObject jsonObject = new JSONObject(capabilities);
```

Comment:
I see what you mean. What I'd like to do in this case is, keep this as is for now since it's not getting merged to `develop` just yet. Once we start using other capabilities, we can re-visit this to make a more conscious decision. Depending on the trade-offs we can go with either of these 3 solutions.